sdk/deploymentmanager/mgmt-v2019-11-01-preview/src/main/java/com/microsoft/azure/management/deploymentmanager/v2019_11_01_preview/implementation/RolloutImpl.java

```diff
@@ -31,12 +31,12 @@
         this.createOrUpdateParameter.withLocation(inner().location());
         this.createOrUpdateParameter.withTags(inner().getTags());
         return client.createOrUpdateAsync(this.resourceGroupName(), this.name(), this.createOrUpdateParameter)
-            .map(new Func1<RolloutInner, RolloutInner>() {
-               @Override
-               public RolloutInner call(RolloutInner resource) {
-                   resetCreateUpdateParameters();
-                   return resource;
-               }
+            .to(new Func1<Observable<RolloutRequestInner>, Observable<RolloutInner>>() {
```

Comment:
Check if `flatMap` compiles or not. I still not figure out what `to` method does...

The point here is that the func need to be called **after** first Observable is complete.